collectors/PropertiesCollector.py

```diff
@@ -0,0 +1,95 @@
+from BaseCollector import BaseCollector
+import logging
+
+logger = logging.getLogger('vrops-exporter')
+
+
+class PropertiesCollector(BaseCollector):
+
+    def __init__(self):
+        super().__init__()
```

Comment:
is this really needed? It is inheriting `__init__` and will initialise ist. 
Question applies to all occurrences of the single super init.

Reply:
Yes, at a high level we want to have access to methods in the superclass. So we need `super()` here. Otherwise it will not work.

Reply:
What you are doing here is an overwrite of `__init__` from the parent class. Then you are calling the `__init__` of the parent/super class and do nothing else. This call is needed, in case you want to enrich init with something. Unless this is done, you can remove `__ init__` here. 

VMPropertiesCollector however still needs it, since it is doing some enrichment, which wouldn't happen otherwise.

Reply:
Alright, got your point. Removed it from `StatsCollector` and `PropertiesCollector`.